drivers/led_strip/ws2812_rpi_pico_pio.c

```diff
@@ -21,7 +22,7 @@ struct ws2812_led_strip_data {
 
 struct ws2812_led_strip_config {
 	const struct device *piodev;
-	uint32_t output_pin;
+	const uint8_t gpio_pin;
```

Comment:
Unfortunately, using the GPIO_DT_SPEC generates dependency on the GPIO module even if not using the GPIO function.
Increasing the code size is very little, but I think reducing the complexity of module dependency should be prioritized.

Reply:
I think it can handle with two modifications.
One is inverting the polarity of the PIO program output.
It makes complex the initialization, but it can.

The other is need to inverting initialize pin state.
It needs to redesign the pinctrl implementation.
It may become a bit large change.

So, I would like to make this a limitation at this point.
I wrote the limitation to the description section of dts.

Reply:
@soburi ^

Reply:
> So, I would like to make this a limitation at this point. I wrote the limitation to the description section of dts.

Thanks for the explanation. It is OK to have limitations. No one complained so far.